src/CoreBundle/private/library/classes/TCMSLogChange.class.php

```diff
@@ -1585,6 +1585,7 @@ public static function AddVirtualNonDbExtension($iLine, $sEntryPoint, $sClassNam
      * @param string $sClassName
      *
      * @throws ErrorException
+     * @throws DBALException
```

Comment:
This changes the method signature which is not allowed in a fix release. Could you wrap the exception in an ErrorException (which I normally wouldn't recommend, but here it's the only annotated exception)?

Reply:
done